conda_recipes/aftereffects-25.0/recipe/build_win.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+set -xeuo pipefail
+
+# The location within $PREFIX where AE will be installed
+mkdir -p $PREFIX/opt
+cp -r $SRC_DIR/aftereffects $PREFIX/opt/
+
+# The version without the update number
+AE_VERSION=${PKG_VERSION%.*}
+
+# See https://docs.conda.io/projects/conda/en/latest/dev-guide/deep-dives/activation.html
+# for details on activation. The Deadline Cloud sample queue environments use bash
+# to activate environments on Windows, so we recommend always producing both .bat and .sh files.
+
+mkdir -p "$PREFIX/etc/conda/activate.d"
+mkdir -p "$PREFIX/etc/conda/deactivate.d"
+
+cat <<EOF > "$PREFIX/etc/conda/activate.d/$PKG_NAME-$PKG_VERSION-vars.bat"
+set "AE_LOCATION=$PREFIX/opt/aftereffects"
+set "AE_VERSION=$AE_VERSION"
+set "PATH=$PREFIX/opt/aftereffects;%PATH%"
+EOF
+cat "$PREFIX/etc/conda/activate.d/$PKG_NAME-$PKG_VERSION-vars.bat"
+
+cat <<EOF > $PREFIX/etc/conda/activate.d/$PKG_NAME-$PKG_VERSION-vars.sh
+export "AE_VERSION=$AE_VERSION"
+export "AE_LOCATION=\$CONDA_PREFIX/opt/aftereffects"
```

Comment:
Silly question, why is this `$CONDA_PREFIX` instead of `$PREFIX`.

Reply:
because the syntax of bash is not the same as windows bat.